src/polyfem/solver/AdjointNLProblem.cpp

```diff
@@ -282,6 +470,14 @@ namespace polyfem::solver
 	{
 		bool need_rebuild_basis = false;
 
+		std::vector<Eigen::MatrixXd> V_old_list;
```

Comment:
same here

Reply:
It only does caching in general, otherwise this is unused. I can refactor to remove but it would make the function longer.